src/meson.build

```diff
@@ -40,9 +40,15 @@ vkBasalt_src = [
 
 x11_dep = dependency('x11')
 
+vulkan_dep = []
```

Comment:
Am I missing something or you don't actually search for system vulkan?

Reply:
Yes, and that is indeed intended.

If you add this, the library will be linked against libvulkan1. Doing this will result in the Vulkan loading the Layers as with any application that links against it. I haven't tested it actually, but I suspect it wouldn't work.

Reply:
Linking to libvulkan would work, but there's no reason to do so.

Reply:
@stephanlachnit Did you use `dependency('vulkan')`? It'll find vulkan-loader, while we need vulkan-headers here. AFAIK it doesn't provide pkgconfig file, thus, can't be found with `dependency()`. Instead, something like `cc.check_header(''vulkan/vulkan.h)`  could be used.

Reply:
> @stephanlachnit Did you use `dependency('vulkan')`? It'll find vulkan-loader, while we need vulkan-headers here. AFAIK it doesn't provide pkgconfig file, thus, can't be found with `dependency()`. Instead, something like `meson.get_compiler('c').check_header('vulkan/vulkan.h')` could be used.

Ah I see, on Debian the loader is shipped with the headers. On Fedora it would work as well because the loader depends on the headers.

Anyway, yes the `check_header` check could be implemented.